chaindexing-postgres/src/repo.ts

```diff
@@ -47,12 +47,20 @@ export class PostgresRepo extends Repo<Pool, Conn> {
       });
   }
 
-  async streamContractAddresses(
-    conn: Conn,
-    streamer: (contractAddresses: ContractAddress[]) => void
-  ) {
-    streamer(await conn.query.chaindexingContractAddressesSchema.findMany());
-    // TODO: Implement serial streaming version
+  streamContractAddresses(conn: Conn) {
+    let currentPage = 0;
+    let offset = 0;
+
+    const limit = 10;
```

Comment:
On second thought, making the limit an optional second parameter is okay. I will update the Rust's SDK.